core/services/ardupilot_manager/mavlink_proxy/test_all.py

```diff
@@ -113,7 +113,12 @@ def test_mavlink_router(valid_output_endpoints: Set[Endpoint], valid_master_endp
     assert mavlink_router.name() == "MAVLinkRouter", "Name does not match."
     assert re.search(r"\d+", str(mavlink_router.version())) is not None, "Version does not follow pattern."
 
-    allowed_output_types = [EndpointType.UDPClient, EndpointType.TCPServer, EndpointType.TCPClient]
+    allowed_output_types = [
+        EndpointType.UDPServer,
+        EndpointType.UDPClient,
+        EndpointType.TCPServer,
+        EndpointType.TCPClient,
+    ]
```

Comment:
We should also test multiple and different orders of endpoints, such as:
`udpin, udpout, udpin, tcpout, tcpin`
`udpout, udpin, tcpout, tcpin, udpin`
`udpin, tcpout, tcpin, udpin, udpout` and etc